src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import Btn from './btn';
+
+// eslint-disable-next-line import/extensions
+import Output from './output.js';
```

Comment:
Your app is throwing an error due to wrong naming of the import folder `./output.js` instead of the `./Output.js` as shown in the display below. 👇
Kindly check and rectify this anomaly and also make sure the quotes are properly displayed. 😉

![Screenshot from 2023-03-23 22-40-14](https://user-images.githubusercontent.com/65233860/227380714-e2e7ebc6-ec08-483c-8699-e10837065c31.png)